3.9-minimal/Dockerfile.c9s

```diff
@@ -47,7 +47,7 @@ LABEL summary="$SUMMARY" \
       io.openshift.expose-services="8080:http" \
       io.openshift.tags="builder,python,python39,python-39,rh-python39" \
       com.redhat.component="python-39-container" \
-      name="sclorg/python-39-minimal" \
+      name="sclorg/python-39-c9s-minimal" \
```

Comment:
1. You are right for rhel7 and centos7, ubi8 and 9 have it at the beginning before the first slash. From the point of view that we're adding a new component to the name, I'd add it as the last one.
2. It does not matter because all minimal Python containers are and always will be based on the minimal base containers. And from my point of view, it does not make sense to build a full container image on a minimal base image or vice versa. That's the reason I was waiting for the minimal C9S container to become available.

Reply:
For what its worth, Red Hat naming policies for container images say the platform should always be a suffix to the name of the image if not already present in the namespace (like in the case of ubiX/ and rhelX/) images. So I would advocate for `python-39-minimal-c9s` as well.

Reply:
Also the repository I've created on quay.io is named `python-39-minimal-c9s` because that is also the image name you specified in the build and push action. This can be changed though.

Reply:
The name is fixed to the final version `python-39-minimal-c9s`. Thank you all for the inputs!